Source/carthage/Bootstrap.swift

```diff
@@ -26,7 +26,19 @@ public struct BootstrapCommand: CommandType {
 					carthage.println(formatting.bullets + "No Cartfile.resolved found, updating dependencies")
 					return project.updateDependencies(shouldCheckout: options.checkoutAfterUpdate)
 				}
-
+				
+				if let depsToUpdate = options.dependenciesToUpdate {
+					let resolvedCartfileContents = try! String(contentsOfURL: project.resolvedCartfileURL, encoding: NSUTF8StringEncoding)
+					let resolvedCartfile = ResolvedCartfile.fromString(resolvedCartfileContents).value!
+					let resolvedDependencyNames = Set<String>(resolvedCartfile.dependencies.map { $0.project.name })
+					
+					for dep in depsToUpdate {
+						if !resolvedDependencyNames.contains(dep) {
+							return SignalProducer<(), CarthageError>(error:CarthageError.UnresolvedDependency(dep))
```

Comment:
I'm afraid I'm very new to ReactiveCocoa and will need some help with both this and https://github.com/Carthage/Carthage/pull/1294#discussion_r62280905 above. Thanks!